.github/workflows/_release.yml

```diff
@@ -216,6 +216,7 @@ jobs:
           MONGODB_ATLAS_URI: ${{ secrets.MONGODB_ATLAS_URI }}
           VOYAGE_API_KEY: ${{ secrets.VOYAGE_API_KEY }}
           UPSTAGE_API_KEY: ${{ secrets.UPSTAGE_API_KEY }}
+          UPSTAGE_DOCUMENT_AI_API_KEY: ${{ secrets.UPSTAGE_DOCUMENT_AI_API_KEY }}
```

Comment:
is this needed? I don't think there are any integration tests for document ai

Reply:
Removed!